requirements.txt

```diff
@@ -1,3 +1,5 @@
+# torch installation
+--extra-index-url https://download.pytorch.org/whl/cu113; sys_platform != "darwin"
```

Comment:
So this is for all platforms except macOS correct? I thought this was windows only.

Reply:
yes that's correct.